app/scripts/lib/router.js

```diff
@@ -98,12 +101,14 @@ define(function (require, exports, module) {
       'settings/display_name(/)': createChildViewHandler(DisplayNameView, SettingsView),
       'signin(/)': createViewHandler(SignInView),
       'signin_complete(/)': createViewHandler(ReadyView, { type: 'sign_in' }),
+      'signin_confirmed(/)': createViewHandler(ReadyView, { type: 'sign_in_confirmed' }),
```

Comment:
Oh, if there is a difference, keep the new route! Let's go with `verify_token_complete` in that case - a bit verbose, but consistent with the other routes, esp if `verify_sign_in` is renamed to `verify_token`.